handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,63 @@
+
+## Responsibilities
+
+You will:
+
+- Facilitate an inclusive and cohesive team environment where everyone is happy, motivated, and productive.
+- Model, teach, and reinforce [our company values](../../../company/values.md) and [our guiding engineering principles](../index.md#guiding-principles).
+- Ensure the team has clear incremental goals that are documented and are always up-to-date.
+- Regularly communicate the team's progress toward their goals as well as changes in team goals to appropriate stakeholders.
+- Support and coach teammates to grow in their careers and fulfill their responsibilities.
+- Grow the team in a sustainable way so that the team can accomplish more over time.
+
+## Qualifications
+
+- You have 2+ years of experience directly managing at least 4 full-time software engineers. We expect you already know how to lead a team.
+- You have done some amount of coding in the past 2 years (for example: at work or on a personal project) to keep your skills up-to-date. We expect that you are able to review and provide valuable feedback on code, architecture, and designs.
+- You are able to clearly communicate your past accomplishments verbally (example: in an interview) and in writing (examples: in your resume, in your answers to our application questions).
+- You have experience breaking up complex technical problems into smaller achievable milestones that the team can iterate on.
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](../../../company/remote/index.md) that is [distributed across the world](../../../company/team/index.md). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](../../people-ops/compensation.md) and [practical benefits](../../people-ops/benefits-and-perks.md) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](../../index.md)
+
+## Interview process
+
+<!-- 1. You [apply here](#todo). --><!-- This is commented out because I will be sending some outbound messages before accepting applications. -->
+
+1. Intro calls
+   - 45m [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) (VP Engineering)
+   - 30m [Tomás Senart](../../../company/team/index.md#tomás-senart) (Engineering Manager, Cloud)
+1. Interviews.
+   - 1h [Engineering leadership](engineering-leadership.md).
+     - The interviewers will be [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) and another Engineering Manager except Tomás.
```

Comment:
I would ideally like to standardize (for this role) who the second interviewer will be so we get consistent signal and calibration across candidates, but that is something I will think about.